project/Versions.scala

```diff
@@ -0,0 +1,5 @@
+object Versions {
```

Comment:
Although, I can imagine that might make sbt version upgrades harder since upgrading sbt would mean upgrading to latest Scala. Not sure if it's a good thing or bad.